.github/ISSUE_TEMPLATE/performance-question.md

```diff
@@ -0,0 +1,29 @@
+---
+name: "\U0001F947 Performance Question"
+about: Performance question about TiDB which is not caused by bug.
+
+---
+
+## Performance Question
+
+- What version of TiDB are you using?
+<!-- You can try `tidb-server -V` or run `select tidb_version();` on TiDB to get this information -->
+
+- What's the difference between the performance you expect and the performance you observe?
+
+- Are you comparing TiDB with some other databases? What's the difference between TiDB and the compared one?
+
+- If it's a specific SQL query, could you please
+    - let us know whether you analyzed the tables involved in the query or not. And how long it is after you ran the last `ANALYZE`.
+    - tell us whether this SQL always runs slowly, or just occasionally.
+    - provide the `EXPLAIN ANALYZE` result of this query if your TiDB version is higher than 2.1, or you can just provide the `EXPLAIN` result.
```

Comment:
```suggestion
    - The `EXPLAIN ANALYZE` result of this query if your TiDB version is higher than 2.1; otherwise, you can just provide the `EXPLAIN` result.
```